app-config.yaml

```diff
@@ -65,7 +65,12 @@ techdocs:
 
 auth:
   # see https://backstage.io/docs/auth/ to learn about auth providers
-  providers: {}
+  environment: development
+  providers:
+    github:
+      development:
+        clientId: ${AUTH_GITHUB_CLIENT_ID}
+        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
```

Comment:
It looks like we will need to include a production configuration for whenever we wish to deploy to an OpenShift cluster.